pkg/test/framework/components/echo/kube/deployment.go

```diff
@@ -327,20 +327,13 @@ spec:
         istio.io/test-vm: {{ $.Service }}
         istio.io/test-vm-version: {{ $subset.Version }}
     spec:
-      # Disable kube-dns, to mirror VM
```

Comment:
Getting rid of this will let the VM resolve k8s hostnames which we don't want (without using Istio's DNS). Could lead us to miss bugs in DNS for VMs.